test/unit/constructor.js

```diff
@@ -169,6 +169,37 @@ describe('constructor', function() {
 			});
 		});
 
+		describe('jsonData', function() {
+
+			it('should default to falsy', function(done) {
+
+				var options = _.extend({}, manager.config);
+
+				sessionStore = new MySQLStore(options);
+				done = _.once(done);
+
+				expect(sessionStore.options.jsonData).to.be.undefined;
+
+				setTimeout(function() {
+					done();
+				}, 30);
```

Comment:
I duplicated the pattern of the custom `done()` function and `setTimeout`, as these new tests were hanging the test suite. I presume the connection is what is hanging the tests, and there's something in that approach to let the connection get closed?